gov2/cloudwatch/README.md

```diff
@@ -1,108 +1,54 @@
-# AWS SDK for Go V2 code examples for Amazon CloudWatch
+# Amazon CloudWatch examples for the AWS SDK for Go (V2)
 
 ## Purpose
 
-These examples demonstrates how to perform several Amazon CloudWatch operations
-using version 2 of the AWS SDK for Go.
+These examples in this directory demonstrate how to perform Amazon CloudWatch
+operations using version 2 of the AWS SDK for Go.
```

Comment:
The examples in this directory show how to perform Amazon CloudWatch
actions using AWS SDK for Go (v2).

Reply:
changed